### ISSUE TITLE
Issues observed with BP-rewrites plugin

### ISSUE THREAD

**Issue by pavanpatil1** [rtCamp/rtMedia#1888]
```
Environment: 
WordPress: v5.9.3
buddypress: v10.2.0
rtMedia: PR #1887 
BP-rewrites: v1.0.0-beta2 
```

- [ ] Uploaded Media not visible on the activity page. 

On the activity page, the uploaded media is not visible. However, the media uploaded from the media page and members' activity page are properly visible but the media uploaded from the activity page is not visible. 

**Steps to reproduce:** 
Navigate to the activity page ex: https://<site-name>/activity and upload the media and observed the issue. 

https://user-images.githubusercontent.com/44057535/163319541-f40ddbec-6c3c-4e1d-afd7-32cf79489654.mp4

---

- [ ] Media comments added on the activity page are not visible. 

**Steps to reproduce:** 
Navigate to the activity page ex: https://<site-name>/activity and add a comment on media. 

https://user-images.githubusercontent.com/44057535/163319739-a0134bd0-9977-43fd-a2f4-167bf6ca0aa3.mp4

---

- [ ] Media page is not visible in groups. 

In groups, the media page is not visible. A blank page is displayed. 

**Steps to reproduce:** 
1. Create a group and enable media in the group from rtMedia -> BuddyPress settings. 
2. Navigate to the groups page. Ex: <site-name>/groups/<group-name>/media

https://user-images.githubusercontent.com/44057535/163320348-36ed9b05-034d-4cd7-8b36-b9e729c21fc6.mp4

---

- [ ] Not able to upload the media and comments on the group activity page. 

**Steps to reproduce:** 
1. Create a group and enable media in the group from rtMedia -> BuddyPress settings. 
2. Navigate to the groups activity page. Ex: <site-name>/groups/<group-name>/
3. Upload the media and observe the issue. 

https://user-images.githubusercontent.com/44057535/163320786-aa780b87-9bd2-4675-ac4a-720999d0efbd.mp4

---

**Comment by vishalkakadiya** [comment]
@pavanpatil1 These issues are fixed now, thanks!

**Comment by pavanpatil1** [comment]
Closing this issue as fixed in #1887
